### PR TITLE
Changed project structure and added jar with dependencies

### PR DESCRIPTION
**Pull request by IvanKhomka**
Changed project structure and added jar with dependencies, also changed BusTicketService class structure.

### CODE REVIEW

File: src/main/java/andersen/BusTicketService.java [hunk 1]
```diff
@@ -54,7 +72,28 @@ public void validateTickets() {
...
+            String line;
+            while ((line = reader.readLine()) != null) {
+                parseAndValidateTickets(line);
+            }
+        } catch (IOException e) {
+            System.err.println("Error reading file: " + e.getMessage());
+        }
+    }
 
+    private List<BusTicket> tickets;
```
Comment (Alexmansar) [inline src/main/java/andersen/BusTicketService.java:96]: move field before the methods

File: src/main/java/andersen/BusTicketService.java [hunk 2]
```diff
@@ -54,7 +72,28 @@ public void validateTickets() {
...
+    public void readTicketsFromFile(String filename) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(filename))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                parseAndValidateTickets(line);
+            }
+        } catch (IOException e) {
+            System.err.println("Error reading file: " + e.getMessage());
+        }
+    }
```
Comment (Alexmansar) [inline src/main/java/andersen/BusTicketService.java:94]: could be separate class

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@IvanKhomka Please, fix small comments, and also, send pom.xml

File: src/main/java/andersen/BusTicketService.java (hunk 1 above)
Comment (IvanKhomka) [reply src/main/java/andersen/BusTicketService.java]: moved field

File: src/main/java/andersen/BusTicketService.java (hunk 2 above)
Comment (IvanKhomka) [reply src/main/java/andersen/BusTicketService.java]: it should be separate class in busticketservice class or new class?

Comment (Alexmansar) [reply src/main/java/andersen/BusTicketService.java]: new class, not inner)

Comment (IvanKhomka) [reply src/main/java/andersen/BusTicketService.java]: Added new class

**Review by Alexmansar — APPROVED** [review]
@IvanKhomka Could be merged

**Pull request merged**